gulpfile.babel.js

```diff
@@ -243,56 +186,29 @@ gulp.task('scripts', ['lint'], () => {
       sourceMapIncludeSources: true
     }))
     .pipe($.header(banner, {pkg}))
-    .pipe($.concat('material.min.js'))
+    .pipe($.rename({extname: '.min.js'}))
     // Write Source Maps
     .pipe($.sourcemaps.write('.'))
-    .pipe(gulp.dest('dist'))
-    .pipe($.size({title: 'scripts'}));
-});
-
-// Clean Output Directory
-gulp.task('clean', () => del(['dist', '.publish']));
```

Comment:
Clean is useful to wipe away any lingering files left over from a file (i.e. file1.scss creates file1.css, rename to file2.scss which results in file2.css and file1.css in dist).

But I've only seen clean used on an initial run of gulp, i.e. it's not called from a watch task.